src/slotmap.js

```diff
@@ -162,6 +162,7 @@ function visit(slot, baseSlot, obj, path, map) {
         );
     } else if (!label.startsWith('__gap')) {
         const prop = obj[toSnakeCase(label)];
+        if (prop === undefined) return;
```

Comment:
We should remove this, or even better, we should convert it to an `assert` so we don't mask any error.

Reply:
Test started failing after changing this to use `assert`

Reply:
Perhaps, we could only log an error and return safely from `visit`? In this way, we wouldn't break the application and we would still have the error logs for visibility.